### PR TITLE
feature(dcosPackage) Import with id implemented

### PR DESCRIPTION
**Pull request by sveba**
dcos_package resource with import implementation
Example:
terraform import dcos_package.foobar "kafka:kafka-project1"

### CODE REVIEW

File: dcos/resource_dcos_package.go [hunk 1]
```diff
@@ -647,7 +669,7 @@ func resourceDcosPackageRead(d *schema.ResourceData, meta interface{}) error {
 		// Carry the checksum from the previous ID
 		parts := strings.Split(d.Id(), ":")
 		if len(parts) < 3 {
-			d.SetId("")
+			d.SetId()
```
Comment (fatz) [inline dcos/resource_dcos_package.go:672]: SetId needs an argument

Comment (sveba) [reply dcos/resource_dcos_package.go]: Sure, sorry. Strange how this typo could happen.

**Comment by sveba** [comment]
Any chance for merge anytime soon?

**Comment by fatz** [comment]
thanks for bringing this back to my attention. I guess I can spent a bit time on this this week.

I'll plan to do a small test tomorrow and if that goes well I'll merge it and tag the next release

File: dcos/resource_dcos_package.go [hunk 2]
```diff
@@ -59,7 +62,7 @@ func resourceDcosPackage() *schema.Resource {
 			"sdk": {
 				Type:        schema.TypeBool,
 				Optional:    true,
-				Default:     false,
+				Default:     true,
```
Comment (fatz) [inline dcos/resource_dcos_package.go:65]: I don't think this is a good idea to be set by default. If being set on non SDK packages errors will occur. At the moment we tent to leave it to false

**Comment by sveba** [comment]
I agree, but your documentation says defaults to true
https://mesosphere.github.io/terraform-provider-dcos/docs/resources/dcos_package/#sdk

**Comment by fatz** [comment]
haha indeed... you're right... I think changing the docs might be less destructive... However docs are generated... so I'll need to find out why the docs state a different default then the code

**Comment by fatz** [comment]
ah ok found it... we changed that... its manually set. So yeah I'll change the docs

File: dcos/resource_dcos_package.go [hunk 3]
```diff
@@ -596,33 +612,37 @@ func resourceDcosPackageRead(d *schema.ResourceData, meta interface{}) error {
...
-	if !appIdok {
-		log.Printf("[WARN] Missing 'app_id'. Assuming the service is not installed")
+	appId, err := dcosPackageParseID(d.Id())
+
+	if err != nil {
 		d.SetId("")
-		return nil
+		return err
 	}
-	appId := stripRootSlash(vString.(string))
```
Comment (fatz) [inline dcos/resource_dcos_package.go:607]: IMO this is still needed as `/foo` and `foo` are the same exact appId

Comment (sveba) [reply dcos/resource_dcos_package.go:607]: It still happens in dcosPackageParseID() - line 104

File: dcos/resource_dcos_package.go (hunk 2 above)
Comment (sveba) [reply dcos/resource_dcos_package.go]: Reverted to false

**Pull request merged**